### ISSUE TITLE
View menu for sidebar option can be out of sync with sidebar state 

### ISSUE THREAD

**Issue by evanhorn** [JabRef/jabref#8115]
### JabRef version

Latest development branch build (please note build date below)

### Operating system

GNU / Linux

### Details on version and operating system

Ubuntu 20.04, Gnome desktop

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. Select one of the sidebar options (i.e., Groups interface) from the View menu (alternatively, use the shortcut key)  
2. Close the sidebar by clicking on the “X” in the top right corner
3. Open View menu and observe that the sidebar option has a checkmark next to it.
4. Select same sidebar option (with checkmark)
5. Open View menu and observe that the sidebar option checkmark has disappeared even though the sidebar option is active.

**Comment by ThiloteE** [comment]
Can reproduce this on

JabRef 5.4--2021-08-21--644e48d
Windows 10 10.0 amd64 
Java 16.0.2 
JavaFX 16+8

and

JabRef 5.4--2021-09-20--a91420d
Windows 10 10.0 amd64 
Java 16.0.2 
JavaFX 17+18

**Comment by Siedlerchr** [comment]
Might also be related to  #8082

**Comment by evanhorn** [comment]
I don't think it is directly related to #8082. I observed the behavior detailed in #8082 after I submitted this issue, but I have not observed it for any of the previous builds. However, I have observed the behavior that I described for some time and in numerous previous versions. Since it was more cosmetic than an interference (and I was lazy), I didn't submit an issue until I got on here to report a more serious issue.
